Wrappers/Python/test/test_io.py

```diff
@@ -273,3 +273,34 @@ def test_tiff_stack_ImageData_wrong_file(self):
         except:
             assert True
 
+    def test_TIFF_compression(self):
+        self.TIFF_compression_test(8)
+        self.TIFF_compression_test(16)
+        with self.assertRaises(ValueError) as context:
+            self.TIFF_compression_test(12)
+
+    def TIFF_compression_test(self, compression):
+        ig = ImageGeometry(voxel_num_x=4, voxel_num_y=5, voxel_num_z=6)
+        data = ig.allocate(0)
+        data.fill(np.arange(4*5*6).reshape(ig.shape))
+
```

Comment:
could also add test for 4D data, as TIFF writer also deals with this

Reply:
done